docs/imviz/import_data.rst

```diff
@@ -99,7 +99,7 @@ object, you can load it into Imviz as follows:
     from jdaviz import Imviz
 
     # mydatamodel is a jwst.datamodels object
-    ndd = NDData(np.array(mydatamodel.data), wcs=mydatamodel.get_fits_wcs())
+    ndd = NDData(mydatamodel.data, StdDevUncertainty(mydatamodel.err), mydatamodel.dq, mydatamodel.meta.wcs, meta=dict(mydatamodel.meta.items()))
```

Comment:
```suggestion
    ndd = NDData(mydatamodel.data, uncertainty=StdDevUncertainty(mydatamodel.err), mask=mydatamodel.dq, wcs=mydatamodel.meta.wcs, meta=dict(mydatamodel.meta.items()))
```

If we want to be pedantic, only `data` should be positional according to https://github.com/astropy/astropy/blob/8b4bea60151f37c8e19fafb7904d6eff0707fe80/astropy/nddata/nddata.py#L131-L141